src/Middleware/HttpsPolicy/src/HstsServicesExtensions.cs

```diff
@@ -11,6 +11,16 @@ namespace Microsoft.AspNetCore.Builder;
 /// </summary>
 public static class HstsServicesExtensions
 {
+    /// <summary>
+    /// Adds HSTS services.
+    /// </summary>
+    /// <param name="services">The <see cref="IServiceCollection"/> for adding services.</param>
+    /// <returns></returns>
+    public static IServiceCollection AddHsts(this IServiceCollection services)
```

Comment:
This one is actually pointless since AddHsts doesn't do anything except configure HstsOptions. We must have missed that during API review.